### ISSUE TITLE
Build failing if executed from locally installed version inside node_modules

### ISSUE THREAD

**Issue by brianjorden** [vuejs/vuepress#53]
This is a bit odd and I tried to spend a bit of time at least narrowing down a simple and reproducible scenario and "workaround" that I hope might help tracking things down. I seemed to have varying results when testing things, but I have narrowed things down to there being some issue with where the `vuepress.js` file is actually located when it is executed.

After assembling these screenshots, I tried installing globally and still got the same errors using that method. Although, I use a non-standard npm global directory location, so that could and occasionally does cause other issues.

On the surface this looks the same as #34, but hopefully this issue and method is a little easier to reproduce if it is the same. The screenshots include all steps for setup, multiple ways of trying to run the build process, and the somewhat surprising workaround.

> In short, if `vuepress` is executed while under the `node_modules` directory, the build fails. If I simply move the `vuepress` directory out of there and into another directory, even at the same relative depth, it seems to work fine.

I am going to try and dig through the actual code a bit this evening, but wanted to share this first once I had a clearly reproducible scenario. As a side note, I've been extremely happy with what I think this is going to help me a ton with on some internal documentation and really excited to keep playing with it more, so please know the work is appreciated.

![vp1-screenshot from 2018-04-15 14-34-18](https://user-images.githubusercontent.com/12365487/38782144-b610d59a-40bc-11e8-8b0c-4f340edecaf4.png)

![vp2-screenshot from 2018-04-15 14-34-44](https://user-images.githubusercontent.com/12365487/38782153-cd46da52-40bc-11e8-9923-218a39e37fd3.png)

![vp3-screenshot from 2018-04-15 14-35-04](https://user-images.githubusercontent.com/12365487/38782154-cf2c64f4-40bc-11e8-9fe5-7511ffca578a.png)

![vp4-screenshot from 2018-04-15 14-35-17](https://user-images.githubusercontent.com/12365487/38782155-d133399e-40bc-11e8-9de4-c41e143d7e51.png)

![vp5-screenshot from 2018-04-15 14-46-29](https://user-images.githubusercontent.com/12365487/38782157-da40bfa2-40bc-11e8-8a1b-5f4dc9ebf3ee.png)

![vp6-screenshot from 2018-04-15 14-45-13](https://user-images.githubusercontent.com/12365487/38782160-dc2d1bd0-40bc-11e8-960f-9d06dedcf27c.png)

**Comment by brianjorden** [comment]
Realized I should also include the copy/paste friendly and concise versions as well.

```
mkdir vuepress-test
cd vuepress-test
npm init -y
npm install -D vuepress
mv node_modules/vuepress/docs docs
./node_modules/vuepress/bin/vuepress.js build docs
```

I'm still not clear about what ends up being different here, but it does seem to work.

```
mkdir same_level
mv node_modules/vuepress same_level/vuepress
./same_level/vuepress/bin/vuepress.js build docs
```

**Comment by brianjorden** [comment]
Actually I had this same issue yesterday when I was testing on v0.3.0, but those screenshots were from (and show in the first one) the latest v0.3.1 version. Happy to test things if I can though, but don't think you actually meant to try the lower version, can do so if you'd like though.

**Comment by yyx990803** [comment]
Should be fixed with 42f63a8
